js/collections/Moderators.js

```diff
@@ -61,8 +61,11 @@ export default class extends Collection {
         // don't add excluded ids or your own id
         const notExcluded = excludeList.indexOf(mod.id) === -1 && mod.id !== app.profile.id;
         // don't add if not a mod or the mod data is missing
+        // don't add if the currency doesn't match the network
+        const cur = !!app.serverConfig.testnet ? 'tBTC' : 'BTC';
```

Comment:
Could you use `app.serverConfig.cryptoCurrency` to determine the currency as opposed to the testnet flag? That would make it more robust once we start handling other crypto currency values.